src/components/paniniCreator/formSections/formSection/formSectionTypes.d.ts

```diff
@@ -28,7 +27,8 @@ type RemovalsProps = {
   isActive: boolean;
   maxElements: number;
   defaultVal: string;
-  formElementsValues: string[];
-  setFormElementsValues: React.Dispatch<React.SetStateAction<string[]>>;
+  fieldsCurrentLength: number;
+  append: UseFieldArrayAppend<FieldValues, string>;
```

Comment:
**nitpick:** Would be good to keep common naming convention for handlers, like `onAppend`, `onRemove` to make it clear that it is. Btw, I think I would not complicate this with such lib specific type